pkg/controller/deployment/util/deployment_util.go

```diff
@@ -362,27 +680,6 @@ func countAvailablePods(pods []api.Pod, minReadySeconds int32) int32 {
 	return availablePodCount
 }
 
-func IsPodAvailable(pod *api.Pod, minReadySeconds int32, now time.Time) bool {
```

Comment:
Sorry about that @kargakis , I seem to have messed up the function ordering, I'll sort it out.